packages/botonic-react/src/utils.js

```diff
@@ -126,3 +126,15 @@ export const getWebchatElement = () =>
 export const setWebchatElementHeight = newHeight => {
   getWebchatElement().style.height = newHeight
 }
+
+export const serializeRegexs = (_, value) => {
+  if (value instanceof RegExp) return value.toString()
+  return value
+}
+
+export const rehydrateRegex = regexStr => {
```

Comment:
I'd prefer the name deserialize for 2 reasons:
1) to be consistent with "serializeRegexs"
2) [hydration](https://stackoverflow.com/a/6991192/145289) means adding complementary data from a database (imagine you serialize only the ids, the hydration means fetching the models from the db which have these ids)